spec/controllers/api/v1/jobs/create_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+describe API::V1::Jobs::Create, type: :request do
+  let(:endpoint) { "/api/v1/jobs/" }
+
+  subject { post endpoint, params: params  }
+
+  context 'with valid params' do
+    let(:params) { attributes_for(:job) }
+
+    it 'has 201 status code' do
+      subject
+      expect(response).to have_http_status :created
+    end
+
+    it 'should be one more job' do
```

Comment:
Sometimes you use `should` and sometimes you don't. Could you please explain why?